---
layout: default
---
# GB0247

## Graph-Break Type
*Short name describing what triggered the graph break*

Shape mismatch with out= tensor variant

## Context
*Values or code snippet captured at the break point*

fn={self.value}, args={args}, kwargs={kwargs}

## Explanation
*Explanation of why the graph break was triggered*

Shape mismatch when calling {self.value} with `out=`. Provided `out=` shape: {saved_out_shapes}. Actual shape: {fake_out.shape}.

## Hints
*Hints on how to resolve the graph break*

- It may be possible to write Dynamo tracing rules for this code. Please report an issue to PyTorch if you encounter this graph break often and it is causing performance issues.


## Additional Information

<!-- ADDITIONAL INFORMATION START - Add custom information below this line -->

<!-- ADDITIONAL INFORMATION END -->


[Click here to add Additional Info](https://github.com/meta-pytorch/compile-graph-break-site/edit/main/docs/gb/gb0247.md)

[Back to Registry](../index.html)
